extensions/amp-accordion/amp-accordion.md

```diff
@@ -127,79 +133,68 @@ animation when the content is expanded and "roll up" animation when collapsed.
 
 [/example]
 
-[filter formats="websites"]
-
-### disable-session-states
-
-Include the `disable-session-states` attribute on `<amp-accordion>` to disable
-collapsed/expanded state preservation.
-
-[/filter] <!-- formats="websites" -->
-
 ### expanded
 
 Apply the `expanded` attribute to a nested `<section>` to expand that section when the page loads.
 
-### expand-single-section
-
-Apply the `expand-single-section` attribute to `amp-accordion` to specify that
-only one `<section>` can expand at a time. If the user clicks or taps on a
-collapsed `<section>`, any currently expanded `<section>` collapses.
+Bind the `expanded` attribute to a `<section>` to expand or collapse that
+section. An expanded section collapses if the expression evaluates as false. A
```

Comment:
Is there a syntax to just toggle the expand/collapse, or do we have to specify as true or false?

Reply:
Unfortunately not for the bind examples.